sdks/python/apache_beam/utils/annotations.py

```diff
@@ -90,6 +90,8 @@ def inner(*args, **kwargs):
       message = '%s is %s' % (fnc.__name__, label)
       if label == 'deprecated':
         message += ' since %s' % since
+        if extra_message is not None:
```

Comment:
Just because it is shorter? Other than that, I can only think of a 1 (append) operation saving in the event that an empty string is passed.